weekly-updates/weekly-update.2015-08-28.md

```diff
@@ -0,0 +1,69 @@
+* `process.send()` is not synchronous as the docs suggest, a regression introduced in 1.0.2, see [#760](https://github.com/nodejs/io.js/issues/760).
+* Calling `dns.setServers()` while a DNS query is in progress can cause the process to crash on a failed assertion. [#894](https://github.com/nodejs/io.js/issues/894)
+* `url.resolve` may transfer the auth portion of the url when resolving between two full hosts, see [#1435](https://github.com/nodejs/io.js/issues/1435).
+
+
+### We need Node.js v4 evangelists
+
+* Node.js v4 is released soon. We need evangelists for Node.js v4. If you have an interest to evangelize, send your information in [this issue](https://github.com/nodejs/node/issues/2633).
+* Current members:
+- Fedor Indutny ([@indutny](https://github.com/indutny))
+- Yosuke Furukawa ([@yosuke-furukawa](https://github.com/yosuke-furukawa))
+- Sakthipriyan Vairamani ([@thefourtheye](https://github.com/thefourtheye))
+- Jackson Tian ([@JacksonTian](https://github.com/JacksonTian))
+- P.S.V.R ([@pmq20](https://github.com/pmq20))
+- James M Snell ([@pmq20](https://github.com/pmq20))
+- Mikeal Rogers ([@mikeal](https://github.com/mikeal))
+- Colin Ihrig ([@cjihrig](https://github.com/cjihrig))
+- Jeremiah Senkpiel ([@Fishrock123](https://github.com/Fishrock123))
+- Rich Trott ([@Trott](https://github.com/Trott))
+- Julian Duque ([@julianduque](https://github.com/julianduque))
+- Stephen Belanger ([@Qard](https://github.com/Qard))
+- Ben Noordhuis ([@bnoordhuis](https://github.com/bnoordhuis))
+- Trevor Norris ([@trevnorris](https://github.com/trevnorris))
+
+### New nodejs.org went live today
+
+* New nodejs.org went live today, served from DigitalOcean and CloudFlare, contributors should go to [https://github.com/nodejs/new.nodejs.org](https://github.com/nodejs/new.nodejs.org).
+
+### Community Updates
+
+* We *still* need a V8 maintainer for our LTS build! Head on over [to GitHub](https://github.com/nodejs/LTS/issues/28) to see if the requirements match your capabilities.
+* npm posts an blog entry about tips for [testing and deploy](http://blog.npmjs.org/post/127671403050/testing-and-deploying-with-ordered-npm-run-scripts).
+
+If you have spotted or written something about Node.js and io.js, do come over to our [Evangelism team repo](https://github.com/nodejs/evangelism) and suggest it on the [Issues page](https://github.com/nodejs/evangelism/issues), specifically the Weekly Updates issue.
+
+### Upcoming Events
+
+* [NodeConf EU](http://nodeconf.eu/) tickets are on sale, September 6th - 9th at Waterford, Ireland
+* [nodeday](http://nodeday.com/) tickets are on sale, September 10th at Karnataka, India
+* [Node.js Italian Conference](http://nodejsconf.it/) tickets are on sale, October 10th at Desenzano - Brescia, Italy
+* [EmpireNode](http://2015.empirenode.org/), October 23rd at New York, US.
+* [JSConf CO](http://www.jsconf.co/), October 16th - 17th at Ruta N, Medellin
```

Comment:
All the other event listings mention the country they take place in, even if it's explicit in the title of the conference. So maybe mention that this one is in Colombia?